docs/leios-design/README.md

```diff
@@ -546,102 +553,89 @@ instance DSIGNAlgorithm BLS_DSIGN where
 
-2. **Voting Mechanisms:**
-   - Similar sortition-based committee selection
-   - Overlapping vote diffusion requirements
-   - Potential for unified vote aggregation logic
+**Cryptographic infrastructure** offers the most promising near-term synergy. Both protocols are based on the BLS signature scheme using BLS12-381 keys, creating an opportunity for shared cryptographic infrastructure. If key material can be shared across protocols, stake pool operators would need to generate and register only one additional key pair rather than separate keys for each protocol. This shared approach would significantly simplify the bootstrapping process for whichever protocol deploys second.
 
-3. **Priority Scheduling:**
-   - Peras votes also need prioritization over Leios
-   - Shared multiplexer design: Peras > Praos > Leios
-   - Common freshest-first delivery mechanisms
+> [!WARNING]
+> TODO: Reference the respective specifications
 
-### Development Coordination
+The Peras requirement for forward secrecy may necessitate an additional mechanism, but these can be implemented independently of Leios requirements. The proof-of-possession mechanisms required for BLS aggregation are identical across both protocols, allowing for shared implementation and validation procedures.
 
-**Parallel Development Strategy:**
-```
-           Peras                      Leios
-             │                          │
-             ├─── BLS Crypto ───────────┤  (Shared)
-             │                          │
-             ├─── Voting Logic ─────────┤  (Coordinate)
-             │                          │
-             ├─── Network Priority ─────┤  (Coordinate)
-             │                          │
-        (Checkpoint                (Throughput
-         Voting)                    Scaling)
-```
+**Protocol-level interactions** between Leios certified endorser blocks and Peras boosted blocks represent a longer-term research opportunity. In principle, the vote aggregation mechanisms used for endorser block certification could potentially be leveraged for Peras boosting, creating a unified voting infrastructure. However, such integration is likely undesirable for initial deployments due to the complexity it would introduce and the dependency it would create between the two protocols. In the medium to long term, exploring these interactions could yield further improvements to both throughput and finality properties.
 
-**Integration Considerations:**
-- Leios vote production thread can be generalized for Peras votes
-- Network multiplexer handles three priority levels: Peras > Praos > Leios
-- Shared telemetry and monitoring infrastructure
+## Era and hard-fork coordination
```

Comment:
@lehins @disassembler What is your take on upcoming eras and their schedule across hard-forks?